Workspaces/Base/Workspace.cs

```diff
@@ -82,12 +84,7 @@ public Bounds outerBounds
 		/// <summary>
 		/// If true, prevent the resizing of a workspace via the front and back resize handles
 		/// </summary>
-		public bool preventFrontBackResize { set { m_WorkspaceUI.preventFrontBackResize = value; } }
-
-		/// <summary>
-		/// If true, prevent the resizing of a workspace via the left and right resize handles
-		/// </summary>
-		public bool preventLeftRightResize { set { m_WorkspaceUI.preventLeftRightResize = value; } }
+		public bool preventResize { set { m_WorkspaceUI.preventResize = value; } }
```

Comment:
They were both being set to true in Console and Profiler. There is currently no case for just left/right or top/bottom resizing.

Should we bring it back? The main issue with resizing the profiler/console workspaces is that we aren't changing the size of the render texture or the window size, so it doesn't make sense to resize them in any direction.